core/domain/topic_services_test.py

```diff
@@ -240,7 +240,8 @@ def test_delete_subtopic_with_skill_ids(self):
     def test_update_subtopic_skill_ids(self):
```

Comment:
Do another test for update-from-some-other-subtopic.

Do tests for the error cases (invalid changelists).

Reply:
Done.